site/src/pages/TemplateSettingsPage/TemplateSchedulePage/TemplateScheduleForm.tsx

```diff
@@ -404,62 +407,61 @@ export const TemplateScheduleForm: FC<TemplateScheduleForm> = ({
             />
           </Stack>
 
-          <Stack direction="row" alignItems="center">
-            <Checkbox
-              id="allow-user-autostop"
-              size="small"
-              disabled={isSubmitting || !allowAdvancedScheduling}
-              onChange={async () => {
-                await form.setFieldValue(
-                  "allow_user_autostop",
-                  !form.values.allow_user_autostop,
-                );
-              }}
-              name="allow_user_autostop"
-              checked={form.values.allow_user_autostop}
-            />
-            <Stack spacing={0.5}>
-              <strong>Enforce these settings across all workspaces</strong>
-              <span
-                css={{
-                  fontSize: 12,
-                  color: theme.palette.text.secondary,
+          <FormControlLabel
+            control={
+              <Checkbox
+                id="allow-user-autostop"
+                size="small"
+                disabled={isSubmitting || !allowAdvancedScheduling}
+                onChange={async (_, checked) => {
+                  await form.setFieldValue("allow_user_autostop", checked);
                 }}
-              >
-                Workspaces by default allow users to set custom autostop timers.
-                Use this to apply the template settings to all workspaces under
-                this template.
-              </span>
-            </Stack>
-          </Stack>
+                name="allow_user_autostop"
+                checked={form.values.allow_user_autostop}
+              />
+            }
+            label={
+              <StackLabel>
+                Allow users to customize autostop duration for workspaces.
+                <StackLabelHelperText>
+                  Workspaces by default allow users to set custom autostop
+                  timers. Use this to apply the template settings to all
+                  workspaces under this template.
```

Comment:
I'm okay with reverting the checkbox to match the API. Less confusion for users. 

We need to revert this too:

> By default, workspaces will inherit the Autostop timer from this template. Enabling this option allows users to set custom Autostop timers on their workspaces or turn off the timer.